cluster/examples/kubernetes/ceph/cluster-test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: ceph.rook.io/v1
```

Comment:
I liked the `cluster-minimal.yaml` name.

Reply:
`cluster-minimal.yaml` does exist as a separate file and could also be used for testing. It was actually created for the scenario to bootstrap the dashboard and not create OSDs. The basic difference with `cluster-test.yaml` is that it also has an OSD, with placeholders to easily test nodes and devices.

Reply:
Hum ok.